### DISCUSSION TITLE
Flutter android app failed to get resource<title>

### DISCUSSION THREAD

**Discussion by Kage0923** [community/community#42476] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

I am trying to build an app using open street map, and after adding [flutter_osm_plugin](https://pub.dev/packages/flutter_osm_plugin) it hangs on "Running Grade task 'assembleDebug'".

Using flutter run -v I got this:
`Failed to get resource: GET. [HTTP HTTP/1.1 521 : https://jitpack.io/com/github/MKergall/osmbonuspack/6.9.0/osmbonuspack-6.9.0.pom)]`

Is there an alternate way to download the resource and what would it be?

Thanks.

**Comment by TechSavvy6** [comment, marked as answer]
Jitpack servers are down, you can get more updates here: https://github.com/jitpack/jitpack.io/issues/5337

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Kage0923 ! We're happy you're here. I'm glad you were able to get an answer here but, for future reference, you are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this post to the correct category for you. Cheers!
